apps/builder/app/shared/commands-emitter.ts

```diff
@@ -61,11 +61,14 @@ export const createCommandsEmitter = <CommandName extends string>({
       // schedule store.set to the next tick
       // so store.listen is executed after store.set below
       Promise.resolve().then(() => {
-        store.createTransaction([$commandMetas], (commandMetas) => {
-          for (const { handler, ...meta } of commands) {
-            commandMetas.set(meta.name, meta);
+        clientImmerhinStore.createTransaction(
```

Comment:
could be just clientOnlyStore?

Reply:
or clientSyncStore

Reply:
Or: apiSyncstore, canvasSyncStore, pluginSyncStore

Reply:
Wait a second, isn't it that you only want to avoid undo/redo getting synced?

Reply:
Actually now as I think about it I don't understand why you use immerhin for shorthands at all, these things are ephemeral, real persistent things are going to happen in response to shorthands